### ISSUE TITLE
iOS Safari: Multi-touch pinch-zoom on a message shows a mini menu.

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#17763]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open any report.
2. Scroll the message history so the message actor's name is in the center of the screen.
3. Now press 'option+ Click' to show the touch points.
4. Expand the touch points outwards over the message.
5. Notice that the menu will show up.

## Expected Result:
Mini Menu should not show on mobile web or native app.
## Actual Result:
Mini menu is present on mobile web.
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [X] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.3
**Reproducible in staging?:** Needs reproduction
**Reproducible in production?:** Needs reproduction
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 

https://user-images.githubusercontent.com/43996225/233510978-70568804-0118-4e08-bfbb-c700f861b6ac.mov

**Expensify/Expensify Issue URL:**
**Issue reported by:** @parasharrajat 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1681999755272699

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by alexpensify** [comment]
I was OOO yesterday and ran out of time. I'll try to review it again soon.

**Comment by alexpensify** [comment]
Not overdue

**Comment by alexpensify** [comment]
I tried replicating and didn't get the same experience here

![image](https://user-images.githubusercontent.com/8107070/234135803-eee8f2c5-1b2d-4dad-bac6-7f2d2bdb7cca.png)

**Comment by alexpensify** [comment]
I started an internal discussion here: 

https://expensify.slack.com/archives/C049HHMV9SM/p1682378444751169?thread_ts=1681999755.272699&cid=C049HHMV9SM

**Comment by alexpensify** [comment]
I still can't replicate this experience: 

![image](https://user-images.githubusercontent.com/8107070/234710649-6f8fbaf8-9a72-4f41-8439-a349dbd5d46a.png)

**Comment by alexpensify** [comment]
I replied in the thread and will close it tomorrow.

**Comment by alexpensify** [comment]
Thank you @parasharrajat, I'll try to find a physical phone to test with.

**Comment by dylanexpensify** [comment]
I cannot replicate this for the life of me. @parasharrajat is the bug that it shows the emoji selector and highlights your text? This seems extremely edge case

**Comment by alexpensify** [comment]
Following discussions with the team and requesting Dylan's assistance in testing with a physical device, we have been unable to replicate the reported experience. Consequently, we have decided to close this GitHub. Our aim is to encourage users to use the app instead of a mobile browser, hence the reason for closing. We appreciate the heads up, but after much consideration, we have concluded that this is an edge case.
